components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Services/ApplicationRegistry.cs

```diff
@@ -391,6 +392,9 @@ public sealed class ApplicationRegistry : IApplicationRegistry, IApplicationBulk
                 _metrics.TrackValue("applicationsAdded", added);
                 _metrics.TrackValue("applicationsUpdated", updated);
                 _metrics.TrackValue("applicationsUnchanged", unchanged);
+                _appsAdded.Set(added);
```

Comment:
Can we create a metrics logger based on Prometheus (maybe one that sends it to app insights and pometheus)?   Now this looks pretty redundant.